aries-site/src/data/wcag/datatable.json

```diff
@@ -0,0 +1,518 @@
+    "label": "Captions (Live)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.2.5",
+    "label": "Audio Description (Prerecorded)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.2.6",
+    "label": "Sign Language (Prerecorded)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.2.7",
+    "label": "Extended Audio Description (Prerecorded)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.2.8",
+    "label": "Media Alternative (Prerecorded)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.2.9",
+    "label": "Audio-only (Live)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.3.1",
+    "label": "Info and Relationships",
+    "status": "failed",
+    "notes": "sortable and searchable headings need improvement"
+  },
+  {
+    "rule": "1.3.2",
+    "label": "Meaningful Sequence",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "1.3.3",
+    "label": "Sensory Characteristics",
+    "status": "failed",
+    "notes": "sortable and searchable headings need improvement"
+  },
+  {
+    "rule": "1.3.4",
+    "label": "Orientation",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "1.3.5",
+    "label": "Identify Input Purpose",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "1.3.6",
+    "label": "Identify Purpose",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.4.1",
+    "label": "Use of Color",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "1.4.2",
+    "label": "Audio Control",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.4.3",
+    "label": "Contrast (Minimum)",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "1.4.4",
+    "label": "Resize Text",
+    "status": "conditional",
+    "notes": ""
+  },
+  {
+    "rule": "1.4.5",
+    "label": "Images of Text",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.4.6",
+    "label": "Contrast (Enhanced)",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "1.4.7",
+    "label": "Low or No Background Audio",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.4.8",
+    "label": "Visual Presentation",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.4.9",
+    "label": "Images of Text (No Exception)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.4.10",
+    "label": "Reflow",
+    "status": "conditional",
+    "notes": "This really depends on content but the developer almost always must switch representations or hide columns at smaller sizes."
+  },
+  {
+    "rule": "1.4.11",
+    "label": "Non-text Contrast",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "1.4.12",
+    "label": "Text Spacing",
+    "status": "not-applicable",
```

Comment:
I think this rule is applicable

Reply:
Not sure why we ended up not-applicable. I think it would be failed just because there's a way to cause the heading text to overlap the search button in the header if you overquish the table somehow (ie. the box containing the search allows flex-shrink). Normal conditions it's all good spacing

Reply:
ah right remember talking about that I will updated to failed

Reply:
updated